nix/build-dapp-package.nix

```diff
@@ -47,6 +48,7 @@ in
           '')
           passthru.libPaths;
       buildPhase = ''
+        ln -s ${pkgs.writeText "dapprc" dapprc} ./.dapprc
```

Comment:
Instead of this, it could be nice to have the option to pass a `dapprc` explicitly as an argument (to `dapp build` I guess?).

Another (unrelated) nit is that `.dapprc` should probably be (first) looked up in `XDG_CONFIG_HOME`